Simple-Projects/Simple_Projects/Car/Car.cpp

```diff
@@ -1,15 +1,15 @@
+#include <ostream>
 #include "Car.h"
 
-Car::Car(double capacity, double consumption, const Point& location, const std::string& model): 
-        fuelCapacity(60), fuelConsumption(0.6), location(Point(0,0)), model("Mercedes") {
-    this->fuelCapacity = capacity;
-    this->fuelAmount = capacity;
-    this->fuelConsumption = consumption;
-    this->location = location;
-    this->model = model;
+Car::Car(double capacity, double consumption, const Point& location, std::string model): 
```

Comment:
Accepting string by value here will allow us to use move semantics when the caller does not intend to use their string anymore